python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -67,6 +67,34 @@ def test_help_command(self):
         pydoc.render_doc(df.foo)
         pydoc.render_doc(df.take(1))
 
+    def test_dropDuplicates(self):
```

Comment:
```suggestion
    def test_drop_duplicates(self):
```

Reply:
Let;s also add a short comment with a JIRA. See "Pull Request" at https://spark.apache.org/contributing.html

Reply:
add comment on the test referring to JIRA